tools/syncer.md

```diff
@@ -353,89 +371,121 @@ target-table = "order_2017"
     +---------------+-------+
     1 row in set (0.01 sec)
     ```
+    
+    - 结果为空或者为 0，Syncer 无法同步数据。
+    - Syncer `server-id` 与 MySQL `server-id` 不能相同，且必须在 MySQL cluster 中唯一。
 
-2. 检查 Binlog 相关参数
+3. 检查 Binlog 相关参数。
 
-    - 检查 MySQL 是否开启 binlog
-    - 可以用如下命令确认是否开启了 binlog
-    - 如果结果是 log_bin = OFF，需要开启。开启方式请参考[官方文档](https://dev.mysql.com/doc/refman/5.7/en/replication-howto-masterbaseconfig.html)
+    1. 检查 MySQL 是否开启了 binlog。
 
-    ```sql
-    mysql> show global variables like 'log_bin';
-    +--------------------+---------+
-    | Variable_name | Value  |
-    +--------------------+---------+
-    | log_bin             | ON      |
-    +--------------------+---------+
-    1 row in set (0.00 sec)
-    ```
+        使用如下命令确认是否开启了 binlog：
 
-3. 检查 MySQL binlog 格式是否为 ROW
+        ```sql
+        mysql> show global variables like 'log_bin';
+        +--------------------+---------+
+        | Variable_name | Value  |
+        +--------------------+---------+
+        | log_bin             | ON      |
```

Comment:
some SQL execution result is not line alignment

Reply:
fixed